Yosemite/Yosemite/Model/Storage/OrderItem+ReadOnlyConvertible.swift

```diff
@@ -26,6 +26,7 @@ extension Storage.OrderItem: ReadOnlyConvertible {
         total = orderItem.total
         totalTax = orderItem.totalTax
         variationID = orderItem.variationID
+        parent = orderItem.parent != nil ? NSNumber(value: orderItem.parent!) : nil
```

Comment:
Tip: you can use `map` to prevent using the implicitly unwrapped optional
```suggestion
        parent = orderItem.parent.map { NSNumber(value: $0) }
```

Reply:
That's a nice way to handle it, thanks!